client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -134,6 +156,131 @@ ChangePaymentMethod.propTypes = {
 	isFullWidth: PropTypes.bool.isRequired,
 };
 
+const wpcom = wp.undocumented();
+const wpcomGetStoredCards = () => wpcom.getStoredCards();
+
+function ChangePaymentMethodList() {
+	const [ formSubmitting ] = useState( false );
+	const translate = useTranslate();
+
+	const { storedCards } = useStoredCards( wpcomGetStoredCards, false );
+	const { isStripeLoading, stripeLoadingError, stripeConfiguration, stripe } = useStripe();
+
+	const paypalMethod = useCreatePayPal();
+
+	const stripeMethod = useCreateCreditCard( {
+		isStripeLoading,
+		stripeLoadingError,
+		stripeConfiguration,
+		stripe,
+		shouldUseEbanx: false,
+	} );
+
+	const existingCardMethods = useCreateExistingCards( {
+		storedCards,
+		stripeConfiguration,
+	} );
+
+	const paymentMethods = [ paypalMethod, stripeMethod, ...existingCardMethods ].filter( Boolean );
+	const disabled = isStripeLoading || stripeLoadingError;
+
+	return (
+		<CheckoutProvider
+			items={ [] }
+			total={ {
+				amount: { value: 0, currency: 'USD', displayValue: '$0' },
+				id: 'xyzzy',
+				type: 'FAAAKE',
+				label: 'fake thing',
+			} }
+			onPaymentComplete={ () => {} }
+			showErrorMessage={ () => {} }
+			showInfoMessage={ () => {} }
+			showSuccessMessage={ () => {} }
+			paymentMethods={ paymentMethods }
+			paymentProcessors={ {} }
+			isLoading={ false }
+		>
+			<Card className="change-payment-method__content">
+				<QueryPaymentCountries />
+
+				<CheckoutPaymentMethods
+					summary={ false }
+					isComplete={ false }
+					className={ 'change-payment-method__list' }
```

Comment:
`foo={ 'bar' }` is equivalent to `foo="bar"`, and I think sometimes the latter is easier to read since it's more HTML-like, but this is fine too if you'd like to leave it this way.